src/popup/pages/MultisigDetails.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div class="multisig-details">
+    <DetailsItem
+      :label="$t('pages.multisig-details.address')"
+    >
+      <template #value>
+        <div class="address-row">
+          <Avatar
+            class="avatar"
+            :address="mockAddress"
+          />
+          <AddressFormatted
+            :address="mockAddress"
+            :column-count="9"
+            class="text-address"
+          />
+        </div>
+      </template>
+    </DetailsItem>
+
+    <DetailsItem
+      :label="$t('pages.multisig-details.contract-id')"
+    >
+      <template #value>
+        <div class="address-row">
+          <Avatar
+            class="avatar"
+            :address="mockContractId"
+          />
+          <AddressFormatted
+            :address="mockContractId"
+            :column-count="9"
+            class="text-address"
+          />
+        </div>
+      </template>
+    </DetailsItem>
+
+    <div class="row">
+      <DetailsItem
+        class="details-item"
+        :label="$t('pages.multisig-details.version')"
+        value="1.0.0"
+      />
+      <DetailsItem
+        class="details-item"
+        :label="$t('pages.multisig-details.current-nonce')"
+        value="2"
+      />
+    </div>
+
+    <AuthorizedAccounts
+      :accounts-address="addresses"
```

Comment:
using plural form may be a false friend - it's tough to spot the difference between singular and plural - you should use two-word name for it